packages/mermaid/src/diagrams/sequence/sequenceDiagram.ts

```diff
@@ -10,4 +10,7 @@ export const diagram: DiagramDefinition = {
   db,
   renderer,
   styles,
+  init: ({ wrap }) => {
+    db.setWrap(wrap);
+  },
```

Comment:
I'm not familiar with this `wrap` option, what does it do? Is it necessary to have it in `init` function of the diagram definition? I also saw it in `c4`.

Reply:
That's a good point. What does `wrap` even do?

For other diagrams, should we have something like:

```js
init: ({ wrap }) => {
  if (wrap) {
    // or maybe just print a warning?
    throw new Error("This diagram does not support %%{wrap}%%");
  }
},
```

Reply:
I see, but does it has to be inside the DB?
Can't we just use `getConfig`?

Reply:
> Wrap makes the text wrap on messages for instance.

Ah, that makes sense!

FYI, there is a [`MermaidConfig.wrap`](https://mermaid.js.org/config/schema-docs/config.html#wrap) value, but the documentation for it is empty, which is why I got a bit confused!

However, [`MermaidConfig.c4.wrap`](https://mermaid.js.org/config/schema-docs/config.html#wrap-1), and [`MermaidConfig.sequence.wrap`](https://mermaid.js.org/config/schema-docs/config.html#wrap-2) are documented.

---

> I see, but does it has to be inside the DB?
> Can't we just use `getConfig`?

They're currently two separate values (but it's probably okay to refactor this in a future PR).

If somebody does:

```mermaid-example
%%{init: {'sequence': {'wrap': false}}}%%
%%{wrap}%%
sequenceDiagram
    Alice->>+John: Hello John, how are you? I am writing a verrrrrrrrrrrry long text.
```

```mermaid
%%{init: {'sequence': {'wrap': false}}}%%
%%{wrap}%%
sequenceDiagram
    Alice->>+John: Hello John, how are you? I am writing a verrrrrrrrrrrry long text.
```

You'd expect `getConfig().sequence.wrap === false`, because that's what the `%%{init: {'sequence': {'wrap': false}}}%%` bit says.

However, the `%%{wrap}%%` is part of the DB, so it would be `true` in the DB.

To be honest, the only main difference I see is that a website could set `siteConfig.secure = ["wrap"]` to block people from modifying the `wrap` config value in a diagram, then doing  `%%{init: {'sequence': {'wrap': true}}}%%` won't work, but `%%{wrap}%%` will work, but I don't really see why anybody would want to do that.

Anyway, if we do want to move `wrap` out of the DB and just keep it in the `config`, that should be separate PR to keep things simple :)